applications/FluidDynamicsApplication/tests/cpp_tests/test_stokes_symbolic_element.cpp

```diff
@@ -0,0 +1,475 @@
+			perturbation_matrix(1,0) = 0.1; perturbation_matrix(1,1) = 0.2; perturbation_matrix(1,2) = 0.6;
+			perturbation_matrix(2,0) = 0.2; perturbation_matrix(2,1) = 0.3; perturbation_matrix(2,2) = 0.7;
+			perturbation_matrix(3,0) = 0.3; perturbation_matrix(3,1) = 0.4; perturbation_matrix(3,2) = 0.8;
+
+			for(unsigned int i=0; i<4; i++){
+				p_element->GetGeometry()[i].FastGetSolutionStepValue(PRESSURE)    = 0.0;
+				for(unsigned int k=0; k<3; k++){
+					p_element->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY)[k]    = vel_original(i,k);
+					p_element->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY, 1)[k] = 0.9*vel_original(i,k);
+					p_element->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY, 2)[k] = 0.75*vel_original(i,k);
+				}
+			}
+
+			// Compute RHS and LHS
+			Vector RHS = ZeroVector(16);
+			Matrix LHS = ZeroMatrix(16,16);
+
+			p_element->Initialize(); // Initialize the element to initialize the constitutive law
+			p_element->CalculateLocalSystem(LHS, RHS, r_model_part.GetProcessInfo());
+
+			// Compute the error of the perturbation
+			double perturbation = 2e-2;
+			std::vector<double> error_norms;
+			std::vector<double> error_vel_norms;
+			std::vector<double> error_pres_norms;
+
+			for (unsigned int j=1; j<5; ++j)
+			{
+				perturbation /= 2;
+				Vector perturbation_vector = ZeroVector(16);
+
+				for(unsigned int i=0; i<4; i++){
+					for(unsigned int k=0; k<3; k++){
+						p_element->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY)[k] = vel_original(i,k) + perturbation*perturbation_matrix(i,k);
+						perturbation_vector(i*4+k) = perturbation*perturbation_matrix(i,k);
+					}
+				}
+
+				// Compute perturbed RHS and LHS
+				Vector error = ZeroVector(16);
+				Vector RHS_obtained = ZeroVector(16);
+				Vector RHS_perturbed = ZeroVector(16);
+				Vector solution_increment = ZeroVector(16);
+
+				p_element->CalculateRightHandSide(RHS_perturbed, r_model_part.GetProcessInfo());
+
+				solution_increment = prod(LHS, perturbation_vector);
+				noalias(RHS_obtained) = RHS - solution_increment;
+				noalias(error) = RHS_perturbed - RHS_obtained;
+				error_norms.push_back(norm_2(error));
+
+			}
+
+			// Check quadratic convergence
+			for(unsigned int i=1; i<error_norms.size(); ++i) {
+				KRATOS_CHECK_NEAR(error_norms[i-1]/error_norms[i], 2.0, 2.5e-1);
+            }
+
+			// std::cout<<std::endl;
+			// for(unsigned int i=0; i<error_norms.size(); ++i)
+			// 	std::cout << "Error norm "<< i << " " << error_norms[i] << std::endl;
+
+		}
+
+		KRATOS_TEST_CASE_IN_SUITE(ElementSymbolicStokes3D4NStationary, FluidDynamicsApplicationFastSuite)
+		{
+			Model model;
+			ModelPart& r_model_part = model.CreateModelPart("Main", 3);
+
+			// Variables addition
+			r_model_part.AddNodalSolutionStepVariable(BODY_FORCE);
+			r_model_part.AddNodalSolutionStepVariable(PRESSURE);
+			r_model_part.AddNodalSolutionStepVariable(VELOCITY);
+
+			// Process info creation
+			double delta_time = 0.1;
```

Comment:
Why `delta_time` for stationary stokes? What does the stationary mean here? little confused ..

Reply:
In here we check that the steady part of the formulation is correct by intentionally setting the BDF coefficients to zero. As the element implementation is the same, we need to initialize the `delta_time` to avoid a segfault.

Reply:
This means the user of the element has to do the same to perform the steady simulations, that is to set `delta_time` to a non-negative number, and set the BDFs to zero. This is not very intuitive, as BDF and delta time should go hand-in-hand. IMO, 
1. the element should not throw segfault for zero delta time, rather it should be able to handle it.
2. if BDFs are set to zero, `delta_time` should not have an effect on the formulation, but it does through `tau_1`

Moreover, in a steady problem, should one even worry about the time step size?

Reply:
> This means the user of the element has to do the same to perform the steady simulations, that is to set `delta_time` to a positive number, and set the BDFs to zero. This is not very intuitive, as BDF and delta time should go hand-in-hand. IMO,

This can be easily automatized so the user needs to do nothing (this is what we actually did in the  `StructuralMechanicsApplication` GUI). However this is out of the scope of this PR. 

> 
>     1. the element should not throw segfault for zero delta time, rather it should be able to handle it.

All the simulations in Kratos assume there is a delta time different than 0 (I refer here to the base `AnalysisStage`). In case a stationary simulation is solved, this is unitary and we assume a total simulation time of 1.1 so a single time step is solved.

> 
>     2. if BDFs are set to zero, `delta_time` should not have an effect on the formulation, but it does through `tau_1`

We intentionally introduce this into the formulation in all our fluid elements. Of course you are right, as slightly different solutions are obtained for different `delta_time` values for a steady solution. This is why we introduced the `dynamic_tau` parameter (bounded between 0 and 1). Hence, setting `dynamic_tau` to zero effectively deactivates this term, so always the same solution is obtained regardless the time increment. Long story short, such effect on the formulation can be straightforwardly deactivated.

> 
> 
> Moreover, in a steady problem, should one even worry about the time step size?

if `dynamic_tau` is zero the time increment has no influence in the steady case.

Reply:
Alright, the solution is set dynamic tau to 0, keep `delta_time` positive (or rather 1), and assume total simulation time of 1 for steady stokes simulations.  Thank you @rubenzorrilla for the explanation. 😌